### DISCUSSION TITLE
Combined with django

### DISCUSSION THREAD

**Discussion by jet10000** [litestar-org/litestar#991] [Ideas]
I want to use django's ORM and admin , and use it in conjunction with starlite. But there are some problems that I don't know how to solve, How to auto reload it.

https://github.com/tiangolo/fastapi/issues/1495

https://www.uvicorn.org/deployment/#running-programmatically

<img width="986" alt="image" src="https://user-images.githubusercontent.com/2258120/209478778-b723e586-f9fd-4e51-89f7-8e91f5c5df16.png">

<img width="1035" alt="image" src="https://user-images.githubusercontent.com/2258120/209478793-781b4e2e-ce78-4943-bcca-5dfe02c040c0.png">

```
pipenv run python manage.py run_starlite
```

In addition, I think the design of starlite is very similar to the idea of this article

https://dev.to/xba/data-oriented-programming-is-dope-ma2

**Comment by Goldziher** [comment]
Hi there,

Interesting problem. I haven't tried combining Starlite with the Django ORM and frankly have no clue how to do this. As for reloading - you could use [uvicorn reloading with watchfiles](https://www.uvicorn.org/settings/#reloading-with-watchfiles) and watch for file changes.

**Reply by jet10000** [reply]
inspired by this project

https://github.com/gluk-w/django-grpc

https://github.com/gluk-w/django-grpc/blob/master/django_grpc/management/commands/grpcserver.py

In this way, starlite can use django's ORM, which is shown in the screenshots in the above post, and it can already work, but I haven't solved uvicorn's reload=True yet.

**Comment by Goldziher** [comment]
> In addition, I think the design of starlite is very similar to the idea of this article
> 
> https://dev.to/xba/data-oriented-programming-is-dope-ma2

Interesting read! Can you elaborate on what you see as similarities?

**Comment by jet10000** [comment]
This project combines fastapi with django

https://github.com/kigawas/fastapi-django
